config/crd/bases/metal3.io_baremetalhosts.yaml

```diff
@@ -156,7 +156,9 @@ spec:
                 description: A custom deploy procedure.
                 properties:
                   method:
-                    description: Custom deploy method name. This name is specific to the deploy ramdisk used. If you don't have a custom deploy ramdisk, you shouldn't use CustomDeploy.
+                    description: Custom deploy method name. This name is specific
+                      to the deploy ramdisk used. If you don't have a custom deploy
+                      ramdisk, you shouldn't use CustomDeploy.
```

Comment:
These changes should really be in the first commit.

Reply:
oh right